automl/automl-script/readme.md

```diff
@@ -95,14 +99,24 @@ overwritten by the corresponding input in `automl-execution` if this
 is given.
 
 **WARNING** All the fields that appear as `non-preferred` in train,
-validation or test datasets will be considered `non-preferred` in all
+holdout or test datasets will be considered `non-preferred` in all
 the resources created by AutoML. It doesn't matter in which dataset
-(train, validation or test) they are set as non-preferred fields.
+(train, holdout or test) they are set as non-preferred fields.
+
+**WARNING** In order to know your AutoML model performance, you should
```

Comment:
The AutoML script creates partial models and evaluations both in the **RFE** and the **OptiML** selection processes. Those models and evaluations are using samples of the training data and independent validation datasets (extracted from the training data as totally complementary splits to avoid leakage between the Feature Selection and Model Selection steps). Therefore, only the final evaluation of the Fusion model on the holdout data can be used as an estimate for its performance.